src/main/java/dumshenko/daniil/Shape.java

```diff
@@ -0,0 +1,25 @@
+package dumshenko.daniil;
+
+enum ShapeType {
+    CIRCLE,
+    RECTANGLE,
+    SQUARE
+}
+
+public class Shape {
+    ShapeType shapeType;
```

Comment:
package-private? I think it would make sense to have the shapeType property as public.

```java
public ShapeType shapeType;
```

ALWAYS put access modifiers unless you know what you are doing 😉